src/controllers/publish.ts

```diff
@@ -94,6 +94,7 @@ import { SingleLanguageRevision } from '../dtos/single-language/revision';
 import { appConfig } from '../config';
 import { FilterTable } from '../dtos/filter-table';
 import qs from 'qs';
+import { DEFAULT_PAGE_SIZE } from './consumer';
```

Comment:
Might be better to stick this in a separate file that both controllers import, so that we're not importing between controllers, but not a blocker.